sfaira/data/dataloaders/loaders/d10_1038_s41586_019_1289_x/homosapiens_brain_2019_10x3v2_velasco_001.yaml

```diff
@@ -0,0 +1,103 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+        - "GSE129519_expression_11a.6mon.txt.gz"
+        - "GSE129519_expression_GM.6mon.txt.gz"
+        - "GSE129519_expression_HUES66.3mon.txt.gz"
+        - "GSE129519_expression_PGP1.3mon.batch2.txt.gz"
+        - "GSE129519_expression_PGP1.3mon.txt.gz"
+        - "GSE129519_expression_PGP1.6mon.batch3.txt.gz"
+        - "GSE129519_expression_PGP1.6mon.txt.gz"
+dataset_wise:
+    author: "Velasco, Silvia"
+    default_embedding: 
+    doi_preprint: 
+    doi_journal: "10.1038/s41586-019-1289-x"
+    download_url_data:
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5F11a%2E6mon%2Etxt%2Egz"
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5FGM%2E6mon%2Etxt%2Egz"
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5FHUES66%2E3mon%2Etxt%2Egz"
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5FPGP1%2E3mon%2Ebatch2%2Etxt%2Egz"
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5FPGP1%2E3mon%2Etxt%2Egz"
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5FPGP1%2E6mon%2Ebatch3%2Etxt%2Egz"
+        - "https://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129519/suppl/GSE129519%5Fexpression%5FPGP1%2E6mon%2Etxt%2Egz"
+    download_url_meta: "manual,meta_combined.txt,https://singlecell.broadinstitute.org/single_cell/data/public/SCP282/reproducible-brain-organoids?filename=meta_combined.txt"
+    primary_data: True
+    year: 2019
+layers:
+    layer_counts: "X"
+    layer_processed: 
+    layer_spliced_counts: 
+    layer_spliced_processed: 
+    layer_unspliced_counts: 
+    layer_unspliced_processed: 
+    layer_velocity: 
+dataset_or_feature_wise:
+    feature_reference: "Homo_sapiens.GRCh38.84"
+    feature_reference_var_key: 
+    feature_type: "rna"
+    feature_type_var_key: 
+dataset_or_observation_wise:
+    assay_sc: "10x 3' v2"
+    assay_sc_obs_key: 
+    assay_differentiation: 
+    assay_differentiation_obs_key: 
+    assay_type_differentiation: 
+    assay_type_differentiation_obs_key: 
+    bio_sample:
+    bio_sample_obs_key: 
+    cell_line: 
+    cell_line_obs_key: 
+    cell_type: 
+    cell_type_obs_key: "CellType"
+    development_stage: 
+    development_stage_obs_key: "Batch"
+    disease: 
+    disease_obs_key: 
+    ethnicity: 
+    ethnicity_obs_key: 
+    gm:
+    gm_obs_key:
+    individual:
+    individual_obs_key: 
+    organ: "brain"
```

Comment:
we can be a bit more granular for organ as these organoids are patterned dorsal forebrain organoids. I think the most fitting term from the UBERON ontology would be "pallium"

Reply:
done